### PR TITLE
Update NVDIMM feature page

### PR DESCRIPTION
**Pull request by mz-pdm**
Changes proposed in this pull request:

- Escape Bugzilla link.

- Add warning about usage on POWER.

- Add a screenshot.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @mz-pdm 

This pull request needs review by: @ahadas

### CODE REVIEW

**Review by sandrobonazzola — APPROVED** [review]

**Comment by sandrobonazzola** [comment]
Waiting for @ahadas approval for merging

**Review by ahadas — APPROVED** [review]

**Pull request merged**
